tests/test_core.py

```diff
@@ -49,9 +49,9 @@ def decorated(self):
 def bleeding_edge_wasm_backend(f):
   def decorated(self):
     if os.getenv('EMTEST_NO_D8'):
-      unittest.skip('Skipped because EMTEST_NO_D8 is set')
+      self.skipTest('Skipped because EMTEST_NO_D8 is set')
     if not config.V8_ENGINE or config.V8_ENGINE not in config.JS_ENGINES:
-      self.skipTest('only works in d8 for now')
+      self.fail('only works in d8 for now')
```

Comment:
how about: `only works in d8 for now but d8 not found (set EMTEST_NO_D8 to skip these tests)`

Reply:
Alright, it should be good now.